keras_tuner/applications/augment.py

```diff
@@ -36,14 +36,17 @@
 
 
 class HyperImageAugment(hypermodel.HyperModel):
-    """Builds HyperModel for image augmentation.
+    """A image augmentation HyperModel.
```

Comment:
When talking about a generic hypermodel or tuner, don't capitalize the term (e.g. "hypermodel"). When talking about the class, use backticks (e.g. `HyperModel`).